doc/tutorial.rst

```diff
@@ -39,6 +39,12 @@ This tells that when the request arrives to the *http://localhost/foobar* URL,
 it must respond with the provided json. The library accepts here any python
 object which is json serializable. Here, a dict is provided.
 
+.. note::
+
+    It is important to specify what to do with the request (what response to be
+    sent back to the client) otherwise *pytest-httpserver* will error with
```

Comment:
I suggest only to use the more precise subject:
```
    It is important to specify what response to be
    sent back to the client otherwise *pytest-httpserver* will error with
```
Actually, when I specified the response after the request arrived (so I did not defined the response in time),
I received a 500 response with a message that the application crashed or the server is too busy. That was quite misleading.

Reply:
Thanks! I fixed the text.